packages/amplify-cli/src/domain/amplify-toolkit.ts

```diff
@@ -35,6 +35,7 @@ export class AmplifyToolkit {
   private _pushResources: any;
   private _storeCurrentCloudBackend: any;
   private _readJsonFile: any;
+  private _readJsonFromDart: any;
```

Comment:
This is flutter specific I'd not pollute the context and the toolkit if this function is not used from anywhere else, move it to frontend package

Reply:
Done!